main.py

```diff
@@ -1,10 +1,38 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_set = json.load(f)
+    print(players_set)
+
+    for user, info_user in players_set.items():
+
+        race = Race.objects.get_or_create(name=info_user["race"]["name"],
+                                          description=info_user["race"]["description"])
```

Comment:
The `get_or_create` method returns a tuple `(object, created)`. You should unpack this tuple to use the `Race` object correctly in subsequent code. For example: `race, created = Race.objects.get_or_create(...)`.